lib/google/ads/google_ads/resource.rb

```diff
@@ -0,0 +1,1566 @@
+module Google
+  module Ads
+    module GoogleAds
+      def self.resource
+        @resource ||= Module.new do
+        
+          def self.http
+            require "google/api/http_pb.rb"
```

Comment:
I'm not exactly sure why this is getting included. Glancing through the protos in the resources folder I don't see any mentions to this specifically (same for other things from google/api in this file). If we do want to include it for some reason maybe it should go in its own module rather than mixed with Google Ads API stuff.